coderd/notifications/testdata/rendered-templates/smtp/TemplateWorkspaceMarkedForDeletion.html.golden

```diff
@@ -0,0 +1,84 @@
+From: system@coder.com
+To: bobby@coder.com
+Subject: Workspace "bobby-workspace" marked for deletion
+Message-Id: 02ee4935-73be-4fa1-a290-ff9999026b13@blush-whale-48
+Date: Fri, 11 Oct 2024 09:03:06 +0000
+Content-Type: multipart/alternative;  boundary=bbe61b741255b6098bb6b3c1f41b885773df633cb18d2a3002b68e4bc9c4
+MIME-Version: 1.0
+
+--bbe61b741255b6098bb6b3c1f41b885773df633cb18d2a3002b68e4bc9c4
+Content-Transfer-Encoding: quoted-printable
+Content-Type: text/plain; charset=UTF-8
+
+Hi Bobby,

```

Comment:
I didn't know that we could render two content-types. Curious, if we can store golden results in separate files?

Reply:
> Curious, if we can store golden results in separate files?

The premise that I adopted for this PR was one of integration testing. This output is meant to represent exactly what would have been sent to the customer. I did not make them render in the same file. This is how it has been.

To separate them would, in my opinion, introduce artificial opportunity for drift from actual behaviour. These tests are most reliable if they not only mimick, but actually dogfood the actual system's behaviour.

If you'd like me to change it to be multiple files, I would be happy to. Would you mind helping me understand the benefit though?

Reply:
Ok! If there are no benefits, then let's leave them as is. Maybe replace `.html.golden` with `.smtp.golden` as this is not just HTML file, but also Plaintext and headers.

> Would you mind helping me understand the benefit though?
My original intention was keep HTML in `.html.golden` files, and Plaintext in `.txt.golden`